requirements-production.txt

```diff
@@ -1,154 +1,173 @@
-billiard==3.5.0.3
-binaryornot==0.4.4
-bleach==2.1.1
-blinker==1.4
-cchardet==2.1.1
-celery==4.1.0
-certifi==2018.11.29
-chardet==3.0.4
-check-manifest==0.35
-click==7.0
-configparser==3.5.0
-cookiecutter==1.6.0
-coverage==4.4.1
-datacite==0.3.0
-decorator==4.1.2
-docutils==0.14
-dojson==1.3.2
-entrypoints==0.2.3
-enum34==1.1.6
-execnet==1.5.0
-elasticsearch==5.0.0
-elasticsearch-dsl==5.0.0
-Flask==0.12.4
-Flask-Alembic==2.0.1
-Flask-Assets==0.12
-Flask-BabelEx==0.9.3
-Flask-Breadcrumbs==0.4.0
-Flask-CeleryExt==0.3.0
-Flask-Collect==1.2.2
-Flask-Cors==3.0.3
-Flask-Login==0.4.0
-Flask-Markdown==0.3
-Flask-Menu==0.6.0
-Flask-Mistune==0.1.1
-Flask-SQLAlchemy==2.3.2
-Flask-WTF==0.14.2
-fs==0.5.4
-funcsigs==1.0.2
-functools32==3.2.3.post2
-future==0.16.0
-html5lib==1.0b10
-idna==2.7
-imagesize==0.7.1
-invenio-assets==1.0.0b7
-invenio-base==1.0.0a16
-invenio-celery==1.0.0b3
-invenio-config==1.0.0b3
-invenio-db==1.0.0b8
-invenio-i18n==1.0.0b4
-invenio-indexer==1.0.0a10
-invenio-jsonschemas==1.0.0a5
-invenio-pidstore==1.0.0b2
-invenio-previewer==1.0.0a11
-invenio-query-parser==0.6.0
-invenio-records==1.0.0b2
-invenio-records-files==1.0.0a10
-invenio-records-rest==1.0.0b1
-invenio-records-ui==1.0.0b1
-invenio-rest==1.0.0b2
-invenio-search==1.0.0a11
-invenio-search-ui==1.0.0a9
-invenio-theme==1.0.0b2
-ipython==5.5.0
-ipython-genutils==0.2.0
-isort==4.2.15
-itsdangerous==1.1.0
-Jinja2==2.10
-jinja2-cli==0.6.0
-jinja2-time==0.2.0
-jsmin==2.2.2
-jsonpatch==1.16
-jsonpointer==1.12
-jsonref==0.1
-jsonresolver==0.2.1
-jsonschema==2.6.0
-jupyter-client==5.1.0
-jupyter-core==4.3.0
-kombu==4.1.0
-lxml==4.1.0
-Mako==1.0.7
-Markdown==3.0.1
-MarkupSafe==1.1.0
-marshmallow==2.15.1
-mistune==0.7.4
-mock==2.0.0
-msgpack-python==0.4.8
-nbconvert==5.3.1
-nbformat==4.4.0
-node-semver==0.1.1
-ordereddict==1.1
-pandocfilters==1.4.2
-pathlib2==2.3.0
-pbr==3.1.1
-pep8==1.7.1
-pexpect==4.2.1
-pickleshare==0.7.4
-pluggy==0.5.2
-poyo==0.4.1
-prompt-toolkit==1.0.15
-psycopg2==2.7.3.2
-ptyprocess==0.5.2
-py==1.4.34
-pydocstyle==2.1.1
-Pygments==2.2.0
-pymdown-extensions==6.0.0
-pyPEG2==2.15.2
-pytest==3.2.3
-pytest-cache==1.0
-pytest-cov==2.5.1
-pytest-pep8==1.0.6
-python-dateutil==2.6.1
-python-editor==1.0.3
-python-slugify==1.2.4
-pytz==2017.2
-PyYAML==3.12
-pyzmq==16.0.2
-raven==6.5.0
-redis==2.10.6
-requests==2.20.0
-scandir==1.6
-simplegeneric==0.8.1
-simplejson==3.11.1
-six==1.11.0
-snowballstemmer==1.2.1
-speaklater==1.3
-Sphinx==1.6.5
-sphinxcontrib-websupport==1.0.1
-SQLAlchemy==1.1.14
-SQLAlchemy-Continuum==1.3.2
-SQLAlchemy-Utils==0.32.19
-testpath==0.3.1
-traitlets==4.3.2
-typing==3.6.2
-Unidecode==0.4.21
-urllib3==1.24.1
-uWSGI==2.0.15
-uwsgitop==0.10
-vine==1.1.4
-wcwidth==0.1.7
-webargs==1.8.1
-webassets==0.12.1
-webencodings==0.5.1
-Werkzeug==0.14.1
-whichcraft==0.4.1
-WTForms==2.1
-xrootdpyfs==0.1.6
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file=requirements-production.txt
```

Comment:
I suspect I run it inside macOS since I can see the command in my history. Which version of pip-tools did you have to generate this list? With the latest `pip-compile` I get a very different format.

Reply:
Updated using:
```
$ docker-compose exec web bash
[root@2c3de0693db4 code]# pip install pip-tools==5.4.0
[root@2c3de0693db4 code]# requirements-production.txt
```